tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckCLDR.java

```diff
@@ -1117,6 +1119,44 @@ public static boolean hasType(List<CheckStatus> result, Type type) {
             }
             return false;
         }
+
+        /**
+         * @returns true if this status applies to the entire locale, not a single path
+         */
+        public boolean isNonPathBased() {
+            return nonPathBased;
+        }
+
+        /** Mark this CheckStatus as nonPathBased */
+        CheckStatus setNonPathBased() {
+            nonPathBased = true;
+            return this;
+        }
+
+        private boolean nonPathBased = false;
+
+        @Override
+        public int compareTo(CheckStatus o) {
+            if (this == o) return 0;
+            int rc = 0;
+            if (rc == 0) {
```

Comment:
ComparisonChain is more bullet proof. https://guava.dev/releases/21.0/api/docs/com/google/common/collect/ComparisonChain.html

Reply:
I just noticed the `@author` on ComparisonChain, nice